### DISCUSSION TITLE
Reverse motion in cloned axis

### DISCUSSION THREAD

**Discussion by enddDD** [fra589/grbl-Mega-5X#297] [Q&A] [answered]
Hi, i have one more question, i need to have 2 motors on y axis(cnc router), they are belt driven so the motors must spin in oposite directions. How do i do that? Thank you in advance.

**Comment by RaphaelDives** [comment, marked as answer]
Hey @enddDD 
Take a look at https://github.com/fra589/grbl-Mega-5X/wiki/Grbl-Configuration

it is the $3 Parameter. There is a table at the $2 description to look up the right value.
(Of course you can calculate it yourself as well. It’s just a binary conversion 😉)

Cheers
Raphael

**Reply by enddDD** [reply]
Switching the cable connector seems to solve the issue

**Reply by RaphaelDives** [reply]
Yes, that’s also a possibility. 
Forgot about that… 😉
